Binance Detect Moonings.py

```diff
@@ -91,6 +91,23 @@ def flush(self):
 
 sys.stdout = St_ampe_dOut()
 
+def is_fiat():
+    # check if we are using a fiat as a base currency
+    global hsp_head
+    PAIR_WITH = parsed_config['trading_options']['PAIR_WITH']
+    #list below is in the order that Binance displays them, apologies for not using ASC order
+    if (PAIR_WITH == ( 'USDT' or 'BUSD' or 'AUD' or 'BRL' or 'EUR' or 'GBP' or 'RUB' or 'TRY' or 'TUSD' or 'USDC' or 'PAX' or 'BIDR' or 'DAI' or 'IDRT' or 'UAH' or 'NGN' or 'VAI' or 'BVND')):
```

Comment:
This needs to be a list, while it works its completely unreadable.

Reply:
You can do, 

```py
if PAIR_WITH in list:
  do x
```

Reply:
thank you, great advice - corrected to:
`    fiats = ['USDT', 'BUSD', 'AUD', 'BRL', 'EUR', 'GBP', 'RUB', 'TRY', 'TUSD', 'USDC', 'PAX', 'BIDR', 'DAI', 'IDRT', 'UAH', 'NGN', 'VAI', 'BVND']

    if PAIR_WITH in fiats:
        return True`